content/en-us/projects/assets/api.md

```diff
@@ -1,195 +1,283 @@
 ---
-title: External Catalog Queries
-description: External Catalog Queries let you externally query the Creator Store Catalog and Marketplace Catalog.
+title: Creator Store Queries
+description: Creator Store Queries let you externally query the Creator Store and Marketplace Catalog.
 ---
 
-You can search Roblox's assets outside Studio by accessing the external catalog API. Use the [Creator Store API](#creator-store-api) to query Studio assets, such as meshes, models, and audio, and the [Marketplace API](#marketplace-api) to query avatar assets on the Marketplace.
+You can search Roblox's assets outside Studio by accessing the Creator Store API. Use the [Creator Store API](#creator-store-api) to query Studio assets, such as meshes, models, and audio, and the [Marketplace API](#marketplace-api) to query avatar assets on the Marketplace.
 
 Each API requires a URL and custom search parameters for that specific catalog. If both URL and parameters are valid, the API returns a JSON format with the results of your search.
 
 ## Creator Store API
 
+<Alert severity="info">
+The `toolbox-service` API described below is not currently covered on the official [Cloud API](../../cloud/reference/index.md) documentation. Better support for Creator Store search with a cleaner API is underway.
+</Alert>
+
 You can query items from the Creator Store catalog using the following URL:
-`https://search.roblox.com/catalog/json?[params]`
+`https://apis.roblox.com/toolbox-service/v1/marketplace/{categoryId}[params]`
 
-You can replace `[params]` with the appropriate [query parameters](#query-parameters) to customize your search.
+You can replace `[categoryId]` and `[params]` with the appropriate [query parameters](#query-parameters) to customize your search.
+
+Valid category IDs are:
+
+- Audio = `3`
+- Model = `10`
+- Decal = `13`
+- Animation = `24`
+- Plugin = `38`
+- MeshPart = `40`
+- Video = `62`
+- FontFamily = `73`
+- Music = `300`
```

Comment:
This could be a bit confusing, as there are 9 items listed here, then 13 items listed for `categoryId (path)` in the table below. Are these the same thing? Or this a subset of the tabled one, and what qualification makes it a subset?

Reply:
I updated it to only include the 9 items here and below